### PR TITLE
349 improve neon proxy logging to filter a request and the corresponding response by 'method'

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**

### CODE REVIEW

File: proxy/docker-compose-test.yml [hunk 1]
```diff
@@ -65,6 +65,7 @@ services:
       POSTGRES_USER: neon-proxy
       POSTGRES_PASSWORD: neon-proxy-pass
       NEW_USER_AIRDROP_AMOUNT: 100
+      LOG_SENDING_SOLANA_TRANSACTION: "YES"
```
Comment (otselnik) [inline proxy/docker-compose-test.yml:68]: Do we really need this variable?

Comment (vasiliy-zaznobin) [reply proxy/docker-compose-test.yml:68]: It is only for CI. It allows to investigate logs in more detail. So it looks cool for test goals

Comment (otselnik) [reply proxy/docker-compose-test.yml:68]: But it is not used anywhere.

Comment (vasiliy-zaznobin) [reply proxy/docker-compose-test.yml:68]: Yes. It is used only for CI. In other cases this variable is set as NO (by default)

Comment (vasiliy-zaznobin) [reply proxy/docker-compose-test.yml:68]: In testnet/devnet/mainnet we can set this variable as YES for a special proxy for investigating goals only

**Review by otselnik — APPROVED** [review]

**Pull request merged**